map/framework.cpp

```diff
@@ -1923,6 +1923,12 @@ void Framework::DeactivateMapSelection()
   }
 }
 
+void Framework::DeactivateMapSelectionCircle()
+{
+    if (m_drapeEngine != nullptr)
+        m_drapeEngine->DeselectObject();
```

Comment:
```suggestion
  if (m_drapeEngine)
    m_drapeEngine->DeselectObject();
  else
    LOG(LWARNING, "DeactivateMapSelectionCircle is called without initializing the Drape Engine");
```

Reply:
No one will read this log :) Or otherwise we can pollute the whole Framework with it.

Reply:
Would it be better to crash? Logs can be collected in theory using OM settings or ADB.

Reply:
We alredy have this `if (m_drapeEngine)` in `Framework::DeactivateMapSelection()`. I just copied it to a separate method.

Reply:
What crash? "If" is correct here and additional LOG gives you nothing.

Reply:
What is the correct case when Deactivate... method is called but drape is not initialized yet? Isn't it an abnormal situation?